README.md

```diff
@@ -31,7 +31,7 @@ optionally deploy Node Feature Discovery in the cluster. This is enabled via `nf
 ### SR-IOV Network Operator
 Nvidia Network Operator can operate in unison with SR-IOV Network Operator
 to enable SR-IOV workloads in a Kubernetes cluster. We provide a helm chart to be used to optionally
-deploy [SR-IOV Network Operator](https://github.com/k8snetworkplumbingwg/sriov-network-operator) in the cluster.
+deploy [SR-IOV Network Operator](https://github.com/openshift/sriov-network-operator) in the cluster.
```

Comment:
this undo me commit. you rebase it wrong

Reply:
We need to change it in master too https://github.com/Mellanox/network-operator/blame/master/deployment/network-operator/README.md#L34

Reply:
yes please do it on master as well